stock_picking_report_valued_hide_totals/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Stock Picking Report Valued Hide Totals",
+    "summary": """
+        Hide totals in delivery slip.
```

Comment:
Too genérica, better more detailed:

```suggestion
        Hide totals summary in valued delivery slip.
```